### DISCUSSION TITLE
Add desktop safezones/margins option

### DISCUSSION THREAD

**Discussion by LeBrick-Studio** [calamityjoe87/desktopfolder#2] [Ideas]
Hello, thanks for forking this amazing app! Can you please add an option to customize desktop safezones/margins (empty spaces that are not used for placing desktop icons, similar to phone UI safezones)?
![1](https://user-images.githubusercontent.com/93297305/219966652-3884695f-e460-4afc-8f5f-979993cf04dc.png)

It'll solve the issue of stuff overlapping with Dock and other 3rd-party widgets, like it can now:
![2](https://user-images.githubusercontent.com/93297305/219966666-29b45f75-a668-443d-a03d-cf40eb087a98.png)

**Comment by calamityjoe87** [comment]
Hey, thanks for the idea. I'll try to implement this in the future.
